### DISCUSSION TITLE
Custom weapon pack by Crooked

### DISCUSSION THREAD

**Discussion by Crooked-Mind** [redeclipse/base#1194] [Maps and Content Showcase]
weapons pack is a complete replacement of all weapons in the game, and includes:
- custom sword (changed to a chain mace) 
![изображение](https://user-images.githubusercontent.com/61201938/76162519-54761180-6160-11ea-9cdd-1157cb40af8a.png)
- custom shotgun 
![изображение](https://user-images.githubusercontent.com/61201938/76162534-78395780-6160-11ea-8fb9-39e9d24de379.png)
- custom SMG (changed to tommy gun) 
![изображение](https://user-images.githubusercontent.com/61201938/76162550-97d08000-6160-11ea-81a7-85e85b55253b.png)
- custom flamer 
![изображение](https://user-images.githubusercontent.com/61201938/76162558-a9198c80-6160-11ea-92cf-2780f8d99aa2.png)
- custom plasma 
![изображение](https://user-images.githubusercontent.com/61201938/76162569-c2bad400-6160-11ea-83e4-744ab8dcc25f.png)
- custom zapper
![изображение](https://user-images.githubusercontent.com/61201938/76163680-1ed62600-616a-11ea-9201-6b0a6b960093.png)
- custom sniper rifle 
![изображение](https://user-images.githubusercontent.com/61201938/76163344-0d3f4f00-6167-11ea-94f9-d7487ed3a4ce.png)
- custom pistol (changed to mauser c96)
![изображение](https://user-images.githubusercontent.com/61201938/76163377-50012700-6167-11ea-91c0-692851e98911.png)
- custom grenade 
![изображение](https://user-images.githubusercontent.com/61201938/76163389-7a52e480-6167-11ea-81d1-0ecc12df56df.png)
- custom landmine
![изображение](https://user-images.githubusercontent.com/61201938/76163430-d3bb1380-6167-11ea-8f6e-27fbe9e77b9c.png)
- custom rocket launcher
![изображение](https://user-images.githubusercontent.com/61201938/76163452-06650c00-6168-11ea-860f-c7e6e43a83f7.png)

There are 2 variant for installation:
1. unzip the weapon pack at /documents/my games/red eclipse/master
2. delete the weapon folder in the directory of the game itself, and unpack the weapon pack in its place

pack download:
https://drive.google.com/file/d/1J2EWd2vnRly6PDyyokFtxVgEaSCz4adB/view?usp=sharing

_!this is the final version of the weapon pack, I will not update or change it, however you can do it as you like_

<a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img alt="Лицензия Creative Commons" style="border-width:0" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a><br />Это произведение доступно по <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">лицензии Creative Commons «Attribution-NonCommercial» («Атрибуция — Некоммерческое использование») 4.0 Всемирная</a>.

**Comment by IceflowRE** [comment]
Can you specify a license for the models? (*add it to this thread directly for a short overview*)
